web/layouts/index.html

```diff
@@ -1,5 +1,7 @@
 {{ define "main" }}
   {{ partial "components/hero" .Params.hero }}
+  <!-- {{ partial "components/quicklinks" .Params.hero }} -->
```

Comment:
could be dropped, I guess...
```suggestion
```

Reply:
I would like to refactor / restyle it later, and therefore leave it here.